Add Jadu detection

jamesjacobs opened the pull request:
Add Jadu platform detection.

---

**Test websites**:

- https://www.jadu.net/
- https://www.wasatchcounty.gov/
- https://www.runnymede.gov.uk/
- https://www.rochdale.gov.uk/
- https://www.nuneatonandbedworth.gov.uk/
- https://chorley.gov.uk/
- https://www.ribblevalley.gov.uk/
- https://www.hial.co.uk/
- https://www.slough.gov.uk/
- https://www.oxford.gov.uk/ 
- https://www.dartford.gov.uk/

max-ostapenko commented on src/technologies/j.json:
```diff
@@ -17,6 +17,20 @@
+    "cats": [
+      1
+    ],
+    "dom": "footer a[href*='jadu.net']",
```
```suggestion
```
Could we skip this?
It can give false positive results if technology is mentioned but not actually present.

max-ostapenko replied:
```suggestion
    "headers": {"node": "^jadu-web-."},
```

This tech returns quite a unique header, so I would probably rely on this one.

max-ostapenko commented on src/technologies/j.json:
```diff
@@ -17,6 +17,20 @@
+    "description": "Jadu is a commercial Digital Experience Platform focused on customer self-service with accessibility at the core. Websites, Forms, CRM and Case Management written in PHP with integrations to many enterprise systems.",
+    "icon": "Jadu.png",
+    "scriptSrc": [
+      "/site/js/almond\\.min\\.js",
+      "/site/js/util\\.min\\.js",
+      "/widgets/js/widget\\.js"
```
These are quite common paths that are available on a number of origins.
```suggestion
      "/site/js/almond\\.min\\.js",
```

Example: https://www.napleshomes.properties/, https://www.barendonk.nl/

max-ostapenko replied:
If `almond.min.js` duplicates the header, let's keep the header only to get the best results.

jamesjacobs commented:
@max-ostapenko thanks for the feedback. I've removed the `scriptSrc` and link check. I've added the header check, though that header is not present on all Jadu sites, so as a fallback I've added a check for some text which is generally present in the footer on Jadu powered websites.
